docs/howto/labels.md

```diff
@@ -0,0 +1,116 @@
+# How to define labels
+
+**Note**
+The term label may designate several concepts. Please check the
+[`ml:label` definition](/definitions/label).
+
+This documentation page covers the following topics:
+
+- Defining a [`ml:label`](/definitions/label), including a [class](/definitions/class).
+- Defining a label following the general definition "a kind of tag to
+  idendify an object" or identifying tag.
+
+## Defining a `ml:label`
+
+To define a field as being a [`ml:label`](/definitions/label), one must just
+add `ml:label` to its `dataType`s, for example:
```

Comment:
hmmm.. I'm not convinced we should use dataType to represent the fact that a field is a label. We may want something similar to what we did for enumerations, i.e., an isLabel property that can go on a RecordSet or a Field.